src/main/java/io/fabric8/vertx/maven/plugin/utils/MojoUtils.java

```diff
@@ -123,18 +170,24 @@ public void buildVertxArtifact(MavenProject project, MavenSession mavenSession,
         Plugin vertxMavenPlugin = project.getPlugin(VERTX_PACKAGE_PLUGIN_KEY);
 
         if (vertxMavenPlugin == null) {
-            throw new MojoExecutionException("Plugin :"+VERTX_PACKAGE_PLUGIN_KEY
-                    +" not found or configured");
+            throw new MojoExecutionException("Plugin :" + VERTX_PACKAGE_PLUGIN_KEY
+                    + " not found or configured");
         }
 
         executeMojo(
                 vertxMavenPlugin,
-                goal("package"),
+                goal(GOAL_PACKAGE),
```

Comment:
I'm not so much a fan of using constants for a single use, but thats indeed a matter of taste. So you should keep it ;-)